### PR TITLE
UI user auth follow-up work

### LINKED ISSUE #3765: Follow-up: UI User auth
Need to propagate UI user auth changes to other docs.

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #3765 

Summary of changes:
- Added a UI user auth known issue
- Updated deployment, orchestration, and training docs with content for UI user auth

### CODE REVIEW

File: v2.1/known-limitations.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 {% include v2.1/cdc/known-limitations.md %}
 
+### Admin UI may become inaccessible for secure clusters
+
+<span class="version-tag">New in v2.1:</span> Accessing the Admin UI for a secure cluster now requires login information (username and password). This login information is stored in a system range, which is replicated as per the replication factor. If majority of the nodes with the system range replicas go down, the range might lose quorum, causing the users to be locked out of the Admin UI.
```
Comment (vilterp) [inline v2.1/known-limitations.md]: Harsh but unfortunately still true. (Issue: https://github.com/cockroachdb/cockroach/issues/29270)

Nit: I'd prefer "system table", but I guess system tables are stored in "system ranges".

Comment (Amruta-Ranade) [reply v2.1/known-limitations.md]: Yeah..also, "system table replicas" sounds unconventional. "system range replicas" is familiar terminology for our readers.

**Comment by piyush-singh** [comment]
___
*[v2.1/admin-ui-access-and-navigate.md, line 28 at r5](https://reviewable.io/reviews/cockroachdb/docs/3872#-LP6Pezm-XI_rfMjLgG1:-LP6Pezm-XI_rfMjLgG2:btyczeq) ([raw file](https://github.com/cockroachdb/docs/blob/079d07e58efeb5345277c4326d7dde02e7e8f512/v2.1/admin-ui-access-and-navigate.md#L28)):*
> ```
> If majority of the nodes
> ```
Small nit: If _a_ majority of the nodes

**Comment by Amruta-Ranade** [comment]
@jseldess Ready for second review :)

**Pull request merged**